src/Services/Azure/AppConfig/AppConfigService.cs

```diff
@@ -156,6 +158,132 @@ public async Task DeleteKeyValue(string accountName, string key, string subscrip
+        }
+
+        // Update basic properties if provided
+        if (enabled.HasValue)
+        {
+            featureFlagSetting.IsEnabled = enabled.Value;
+        }
+
+        if (description != null)
+        {
+            featureFlagSetting.Description = description;
+        }
+
+        if (displayName != null)
+        {
+            featureFlagSetting.DisplayName = displayName;
+        }
+
+        // Parse and update complex JSON properties
+        await UpdateFeatureFlagJsonProperties(featureFlagSetting, conditions);
+
+        await client.SetConfigurationSettingAsync(featureFlagSetting, cancellationToken: default);
+    }
+
+    private static async Task UpdateFeatureFlagJsonProperties(
+        FeatureFlagConfigurationSetting featureFlagSetting,
+        string? conditions)
+    {
+        // Update conditions (client filters and requirement type)
+        if (!string.IsNullOrEmpty(conditions))
+        {
+            try
+            {
+                var conditionsDoc = JsonDocument.Parse(conditions);
+                var root = conditionsDoc.RootElement;
+                
+                // Handle client_filters
+                if (root.TryGetProperty("client_filters", out var filtersElement) && filtersElement.ValueKind == JsonValueKind.Array)
+                {
+                    featureFlagSetting.ClientFilters.Clear();
+                    
+                    foreach (var filterElement in filtersElement.EnumerateArray())
+                    {
+                        if (filterElement.TryGetProperty("name", out var nameElement))
+                        {
+                            var filterName = nameElement.GetString() ?? string.Empty;
+                            var parameters = new Dictionary<string, object>();
+                            
+                            if (filterElement.TryGetProperty("parameters", out var paramsElement))
+                            {
+                                foreach (var param in paramsElement.EnumerateObject())
+                                {
+                                    parameters[param.Name] = JsonElementToObject(param.Value);
+                                }
+                            }
+                            
+                            featureFlagSetting.ClientFilters.Add(new FeatureFlagFilter(filterName, parameters));
+                        }
+                    }
+                }
+                  // Note: requirement_type handling may require custom logic or may not be directly supported by this SDK version
+            }
+            catch (JsonException ex)
+            {
+                throw new ArgumentException($"Invalid JSON format for conditions: {ex.Message}", nameof(conditions));
+            }
+        }
+        
+        await Task.CompletedTask;
+    }
+    
+    private static object JsonElementToObject(JsonElement element)
```

Comment:
I think we already have something similar in the codebase.

Reply:
Should be good now.